jenkins/common.groovy

```diff
@@ -45,8 +45,7 @@ def isReviewable() {
 def isDeployable() {
   return (IS_DEV_BRANCH ||
           IS_STAGING_BRANCH) &&
-    !env.CHANGE_TARGET &&
-    !currentBuild.nextBuild // if there's a later build on this job (branch), don't deploy
+    !env.CHANGE_TARGET
```

Comment:
I can't find any usages of `CHANGE_TARGET` in the `vets-website` or `devops` repo. 

- [Search results for `CHANGE_TARGT` in `vets-website` repo](https://github.com/department-of-veterans-affairs/vets-website/search?q=change_target)
- [Search results for `CHANGE_TARGT` in `devops` repo](https://github.com/department-of-veterans-affairs/devops/search?q=change_target)

Reply:
Since I can't find any usages of `CHANGE_TARGET`, I assume it will always be empty/undefined.

If that's the case, then I think `!env.CHANGE_TARGET` will always be true.